ui/src/app/components/runs/run-detail/run-detail.component.ts

```diff
@@ -13,13 +13,13 @@
  * limitations under the License.
  */
 
-import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { Component, Input, OnDestroy, OnInit, OnChanges, SimpleChanges } from '@angular/core';
```

Comment:
You can remove imports: OnChanges, SimpleChanges

Reply:
👍🏽